### PR TITLE
Provide whole auth response on login event

### LINKED ISSUE #41: Login event provide "status" only instead of auth "response[Object]"
From documentation (README.md):
image

But we only have the status instead:
https://github.com/adi518/vue-facebook-login-component/blob/f42d8cfce96feea3b2c3e19952ce972533cb2066/packages/vue-facebook-login-component/src/components/Scope.js#L98

Can you please provide the whole auth response object instead of the status only?

### PR DESCRIPTION
**Pull request by M1CK431**
Fix #41

### CODE REVIEW

File: packages/vue-facebook-login-component/src/components/Scope.js [hunk 1]
```diff
@@ -92,10 +92,10 @@ export default {
     },
     async login() {
       const promise = login(this.loginOptions)
-      const { status } = await this.async(promise)
-      if (Sdk.isConnected(status)) {
+      const result = await this.async(promise)
```
Comment (adi518) [inline packages/vue-facebook-login-component/src/components/Scope.js:95]: Let's name it `response`.

**Comment by adi518** [comment]
Great, I'll merge it soon.

File: packages/vue-facebook-login-component/src/components/Scope.js (hunk 1 above)
Comment (adi518) [reply packages/vue-facebook-login-component/src/components/Scope.js:95]: You should also run tests (they are not automated atm).

Comment (M1CK431) [reply packages/vue-facebook-login-component/src/components/Scope.js]: Done.

NB: please add some notes [here](https://github.com/adi518/vue-facebook-login-component#development) about how to contribute, especially about running unit tests.

Comment (adi518) [reply packages/vue-facebook-login-component/src/components/Scope.js]: Sure, good idea. 👍

**Pull request merged**